src/Template/Element/Flash/default.ctp

```diff
@@ -10,4 +10,3 @@ BUTTON;
 }
 
 echo $this->Html->div($class, $message, $params['attributes']);
-?>
```

Comment:
Please add back the closing `?>`. It's omitted only for class files not templates.